picker-starter/src/components/FieldPlugin.vue

```diff
@@ -1,11 +1,14 @@
 <script setup lang="ts">
```

Comment:
Most important changes to pay attention to @demetriusfeijoo:
- the `enablePortalModal`
- the `watch` with the proper plugin setup